catroid/src/main/res/values/styles.xml

```diff
@@ -583,6 +584,10 @@
         <item name="android:textSize">?attr/x_large</item>
     </style>
 
+    <style name="BrickCategory.RecentlyUsed">
+        <item name="android:background">@drawable/brick_selection_background_recently_used</item>
```

Comment:
You have to add the .9.png file for different solutions as well (not just for xhdpi as you did). This is probably why you got lint-warnings in the first place.

Reply:
but if I'm not mistaken, all the other selection brick backgrounds like control, look etc. have just a single solution file right? (in drawable-xhdpi) wouldn't it be redundant to have more for just the recent bricks?

Reply:
I guess you are right. As there are no actual instances of the recently used type (I mean actual bricks with that grayish background) a single solution file should suffice. I am sorry.